README.md

```
@@ -1,3 +1,160 @@
+- [x] 장기판
+    - [x] 9(가로) x 10(세로) 크기의 장기판을 사용한다.
+    - [x] 입출력 요구사항
+        - [x] 현재 장기판 현황을 출력한다.
+        - [x] 각 칸은 ＿ + 띄어쓰기로 이루어진다.
+
+- [x] 상차림
+  - [x] 왼상차림, 오른상차림, 안상차림, 바깥상차림에 따라 초기 기물 배치가 변경된다.
+  - [x] 입출력 요구사항
+    - [x] 초나라, 한나라 순서로 상차림을 입력받는다.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)